docs/monitoring/spm-faq.md

```diff
@@ -571,24 +575,65 @@ The resulting package will contain all relevant info needed for our
 investigation. You can send it, along with short description of your
 problem, to <support@sematext.com> or contact us in chat.
 
+<b>Container setups</b>
+
+At the moment there is no diagnostics script for container setups, so various resources have to be gathered manually.
+
+Two types of agents are running in container setups:
```

Comment:
Two types of agents are running in container setups -> Two types of agents are running in container setups and spawn from the following images: